content/docs/guides/self-hosted/console.md

```diff
@@ -79,3 +79,20 @@ The following are the core environment variables that are required at a minimum.
 | ------------- | ----------- |
 | BITBUCKET_OAUTH_ID| Atlassian Bitbucket OAuth consumer client ID. Used for Bitbucket OAuth signins. |
 | BITBUCKET_OAUTH_SECRET | |
+
+### TLS Environment Variables
+
+| Variable Name       | Description                                                                                                       |
+|---------------------|-------------------------------------------------------------------------------------------------------------------|
+| CONSOLE_TLS_CERTIFICATE | The TLS certificate. The certificate must be supplied in X.509 format and must be PEM encoded.                |
+| CONSOLE_TLS_PRIVATE_KEY | The private key associated with the TLS certificate. The private key must be PEM encoded.                     |
+| CONSOLE_MIN_TLS_VERSION | The minimum version of TLS to use when serving the console (must be in \<major>.\<minor> format, e.g. `1.2`). |
```

Comment:
Is there a default value for this one? If so, we should mention what the value is, and that it's optional.

Reply:
No, there is not a default value. If the value is not set, a minimum version will just not be enforced. To your point though, I think it would be good to call out that it is an optional setting.